lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2271,11 +2273,12 @@ def scatter(self, xs, ys, zs=0, zdir='z', s=20, c=None, depthshade=True,
 
         if c is None:
             c = self._get_lines.get_next_color()
```

Comment:
Ok, I see why (need to handle the `None` case). However, it should have still deferred to the 2d scatter. So, for the merge, have it so that it accepts my changes from v1.5 over what is in master for this portion of the code. My version defers the handling of `None` to 2d scatter, which is the correct course of action.